### ISSUE TITLE
Proposal: SizeAnimation

### ISSUE THREAD

**Issue by marcelwgn** [microsoft/microsoft-ui-xaml#6922]
# Proposal: SizeAnimation

## Summary

As of right now, there really isn't a way to animate Size properties (mainly ArcSegment.Size).

## Rationale

* It's a lot more convenient to animate through a single animation than having to write two animations with slightly more complicated 

## Scope

| Capability | Priority |
| :---------- | :------- |
| Animate from a Size to another Size | Must |
| Behave to Size objects as PointAnimation behaves to Point objects | Should |

## Important Notes
My expectation would be to have it behave similar to PointAnimation.

## Open Questions

N/A

**Comment by ojhad** [comment]
This seems like a reasonable idea, but it is unlikely that we will get to this in the near future

**Comment by chingucoding** [comment]
Fair enough, this is certainly a niche case. If there was a way to implement it myself, I would but I haven't found much information on this.
